checkout-3ds/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
do we need this file if all is commented?

Reply:
Actually we don't need it at the moment, but it's a part of default auto-generated Android library structure, file is referenced in build.gradle config.